roles/barbican/tasks/main.yml

```
@@ -19,7 +19,7 @@
     chart_ref: "{{ barbican_helm_chart_ref }}"
     release_namespace: "{{ barbican_helm_release_namespace }}"
     create_namespace: true
-    kubeconfig: /etc/kubernetes/admin.conf
+    kubeconfig: "{{ kubeconfig_path | default('/etc/kubernetes/admin.conf') }}"
```

`ansible-lint` won't like this in the long term, because it's not using a role variable.

Would you be able to add something to the `defaults/main.yml` for each role (barbican in this example):

```yaml
barbican_helm_kubeconfig: "{{ kubeconfig_path | default('/etc/kubernetes/admin.conf') }}"
```

and then...

```suggestion
    kubeconfig: "{{ barbican_helm_kubeconfig }}"
```

This would allow users that would for some reason want to use different `kubeconfig` for different roles (e..g. multi-region as @fitbeard described) possible.